src/probe_matcher.cpp

```diff
@@ -115,6 +123,7 @@ std::set<std::string> ProbeMatcher::get_matches_for_probetype(
       symbol_stream = get_symbols_from_file_safe(
           tracefs::available_filter_functions());
```

Comment:
If we changed this to get symbols from `bpftrace_.traceable_funcs_`, we could completely avoid addition of the `ignore_kprobes_blacklist` parameter.

Reply:
The method you said is a good idea, but the amount of code may be larger and more complex, because you need to convert FuncsModulesMap to std::istream, add ignore_kprobes_blacklist is more concise, what do you think? Thank you for your review!!

Reply:
Shouldn't be that hard:
```cpp
std::string funcs;
for (auto &mod : bpftrace_.traceable_funcs_)
{
  for (auto &fun : mod->second)
    funcs += fun->second + "\n";
}
return std::make_unique<std::istringstream>(funcs);
```
Or you can do:
```cpp
    funcs += fun->second + " [" + mod->first + "]\n";
```
if you want the module, too (but we're ignoring it for kprobes anyway).

An advantage is that this way, we don't parse `/sys/kernel/tracing/available_filter_functions` twice.

Reply:
@viktormalik Thanks very much, i'm not familiar with C++, I change the code according to your suggestion, please review.